test/server/lib/DocApi.ts

```diff
@@ -3350,6 +3350,14 @@ function testDocApi() {
       /*
         Regression test for old _subscribe endpoint. /docs/{did}/webhooks should be used instead to subscribe
       */
+      const serving: Serving = await serveSomething(app => {
```

Comment:
This separates a docstring from the function it is describing, can you bring them back together? Also the whitespace looks wrong in this area of the code, sorry about that.